lnrpc/rpc.proto

```diff
@@ -1154,6 +1154,8 @@ message Hop {
     int64 amt_to_forward = 3 [json_name = "amt_to_forward"];
     int64 fee = 4 [json_name = "fee"];
     uint32 expiry = 5 [json_name = "expiry"];
+    int64 amt_to_forward_msat = 6 [json_name = "amt_to_forward_msat"];
```

Comment:
You can add `[deprecated=true]` to the satoshi fields, per https://developers.google.com/protocol-buffers/docs/proto3